packages/client-app/src/assets/tailwind.css

```diff
@@ -1,18 +1,78 @@
-/**
- * Put the Tailwind base into the Scalar base cascade layer
- */
 @import 'tailwindcss/base' layer(scalar-base);
-@tailwind components;
-@tailwind utilities;
 
-@layer components {
-  .centered-x {
-    @apply left-1/2 -translate-x-1/2;
-  }
-  .centered-y {
-    @apply top-1/2 -translate-y-1/2;
-  }
-  .centered {
-    @apply centered-x centered-y;
+.scalar-client {
+  @tailwind components;
+  @tailwind utilities;
+  @tailwind variants;
+
+  /** 
+   * Custom utilities to make life easier 
+   * For some reason it was bugging on apply so I just used css for now 
+   * Replicated in components until we figure out how to share  
+   */
+  @layer components {
+    .row {
+      /* @apply flex flex-row; */
+      display: flex;
+      flex-direction: row;
+    }
+    .row-1 {
+      /* @apply flex flex-1 flex-row; */
+      display: flex;
+      flex-direction: row;
+      flex: 1 1 0%;
+    }
+    .col {
+      /* @apply flex flex-col; */
+      display: flex;
+      flex-direction: column;
+    }
+    .col-1 {
+      /* @apply flex flex-1 flex-col; */
+      display: flex;
+      flex-direction: column;
+      flex: 1 1 0%;
+    }
+    .flex-center {
+      /* @apply flex items-center justify-center; */
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+    .centered-x {
+      /* @apply left-1/2 -translate-x-1/2; */
+      left: 50%;
+      --tw-translate-x: -50%;
+      transform: translate(var(--tw-translate-x), var(--tw-translate-y))
+        rotate(var(--tw-rotate)) skewX(var(--tw-skew-x)) skewY(var(--tw-skew-y))
+        scaleX(var(--tw-scale-x)) scaleY(var(--tw-scale-y));
+    }
+    .centered-y {
+      /* @apply top-1/2 -translate-y-1/2; */
+      top: 50%;
+      --tw-translate-y: -50%;
+      transform: translate(var(--tw-translate-x), var(--tw-translate-y))
+        rotate(var(--tw-rotate)) skewX(var(--tw-skew-x)) skewY(var(--tw-skew-y))
+        scaleX(var(--tw-scale-x)) scaleY(var(--tw-scale-y));
+    }
+    .centered {
+      /* @apply centered-x centered-y; */
+      left: 50%;
+      --tw-translate-x: -50%;
+      top: 50%;
+      --tw-translate-y: -50%;
+      transform: translate(var(--tw-translate-x), var(--tw-translate-y))
+        rotate(var(--tw-rotate)) skewX(var(--tw-skew-x)) skewY(var(--tw-skew-y))
+        scaleX(var(--tw-scale-x)) scaleY(var(--tw-scale-y));
+    }
   }
+
+  text-rendering: optimizeLegibility;
```

Comment:
@hwkr do we need these here? Are these client specific or should they come from the main reset?

Reply:
That's a great question, I'd just do whatever the [Tailwind preflight](https://github.com/tailwindlabs/tailwindcss/blob/next/packages/tailwindcss/preflight.css) does.